### ISSUE TITLE
Numpy Array Type Error

### ISSUE THREAD

**Issue by willw1** [piccolomo/plotext#84]
Great package. This works as expected:
```
import plotext as plt
a = [2,4,3,5,6,7,4,5]
plt.scatter(a)
plt.show()
```

But there seems to be a problem with dynamic type assignment (passing a numpy array directly gives the same error).
```
import numpy as np
import plotext as plt
a = np.array([2,4,3,5,6,7,4,5])
plt.scatter(list(a))
plt.show()
```
Error
```
TypeError                                 Traceback (most recent call last)
Input In [4], in <cell line: 1>()
----> 1 plt.show()

File ~/anaconda3/envs/python38/lib/python3.8/site-packages/plotext/_figure.py:286, in figure_class.show(self)
    285 def show(self):
--> 286     self.build()
    287     write(self.canvas)

File ~/anaconda3/envs/python38/lib/python3.8/site-packages/plotext/_figure.py:274, in figure_class.build(self)
    272 self.subplot.add_grid()
    273 self.subplot.add_extra_lines()
--> 274 self.subplot.update_matrix()
    275 self.subplot.add_legend()
    276 self.subplot.add_yaxis()

File ~/anaconda3/envs/python38/lib/python3.8/site-packages/plotext/_subplot.py:330, in subplot_class.update_matrix(self)
    327 x, y, m, c =  (empty, empty, empty, empty) if xymc == [] else transpose(xymc)
    329 y0 = [get_matrix_data([0], ylim[s], self.height_canvas * yfactor[s])[0] for s in range(self.signals)]
--> 330 y0 = [max(0, el) for el in y0]
    331 x0 = [get_matrix_data([0], xlim[s], self.width_canvas * xfactor[s])[0] for s in range(self.signals)]
    332 x0 = [max(0, el) for el in x0]

File ~/anaconda3/envs/python38/lib/python3.8/site-packages/plotext/_subplot.py:330, in <listcomp>(.0)
    327 x, y, m, c =  (empty, empty, empty, empty) if xymc == [] else transpose(xymc)
    329 y0 = [get_matrix_data([0], ylim[s], self.height_canvas * yfactor[s])[0] for s in range(self.signals)]
--> 330 y0 = [max(0, el) for el in y0]
    331 x0 = [get_matrix_data([0], xlim[s], self.width_canvas * xfactor[s])[0] for s in range(self.signals)]
    332 x0 = [max(0, el) for el in x0]

TypeError: '>' not supported between instances of 'NoneType' and 'int'
```
[EDIT: Looks like it doesn't like numpy.int64 data type]

**Comment by piccolomo** [comment]
Hi @willw1,

thanks for reporting. 

Just quickly try `a.tolist()` instead of `list(a)`. It worked for me. The problem is that without `numpy`, `plotext` cannot do it automatically.

Savino

**Comment by piccolomo** [comment]
The issue is that `list()` of a `numpy` array preserves each element type from `numpy`, while `tolist()` converts to actual `int`.

**Comment by piccolomo** [comment]
I close, but feel free to continue discussion or reopen it necessary. 

Thanks for input, Savino

**Comment by piccolomo** [comment]
Hello @willw1,

Issue solved in [version 4.3](https://github.com/piccolomo/plotext) available only on GitHub for now, using:

```
pip install git+https://github.com/piccolomo/plotext
```

Any feedback or double test is appreciated.

Thanks,
Savino
